server/reports/attendance/attendance-endpoint.js

```diff
@@ -0,0 +1,123 @@
+import MindbodyAccess from "../../api-manager";
```

Comment:
Strangely, my version of node rejected this line until I added the .js file suffix . Truly, I don't understand import and the new ES6 stuff. I'll just push that change to your branch and hopefully it works for everyone else, too. Probably, since an import with the js suffix worked in commission-report